client/payment-methods/index.js

```diff
@@ -83,30 +146,59 @@ const PaymentMethods = () => {
 		);
 	};
 
+	const {
+		featureFlags: { upe: initialUPESetting },
+	} = useContext( WCPaySettingsContext );
+	const [ isUpeEnabled, setIsUpeEnabled ] = useIsUpeEnabled(
+		initialUPESetting
+	);
+	const { status } = useContext( WcPayUpeContext );
+
 	return (
 		<>
-			<Card className="payment-methods">
-				<CardBody size={ null }>
-					<PaymentMethodsList className="payment-methods__enabled-methods">
-						{ enabledMethods.map(
-							( { id, label, description, Icon } ) => (
-								<PaymentMethod
-									key={ id }
-									Icon={ Icon }
-									onDeleteClick={
-										1 < enabledMethods.length
-											? handleDeleteClick
-											: undefined
-									}
-									id={ id }
-									label={ label }
-									description={ description }
-								/>
-							)
-						) }
-					</PaymentMethodsList>
+			{ 'error' === status && <UPEDisableError /> }
+			<Card
+				className={ classNames( 'payment-methods', {
+					'loading-placeholder': 'pending' === status,
+				} ) }
+			>
+				{ isUpeEnabled && (
+					<CardHeader
+						size={ null }
+						className="payment-methods-header"
+					>
+						<PaymentMethodsDropdownMenu
+							isUpeEnabled={ isUpeEnabled }
```

Comment:
This section is conditionally displayed based on `isUpeEnabled` - passing the `isUpeEnabled` prop down is not needed, it will always be `true`.

Reply:
Good point - I passed it down to use it for the `isDisabled` prop but that's unnecessary now that we won't show this header area at all unless UPE is enabled.